contracts/src/v0.8/functions/dev/v1_X/libraries/ChainSpecificUtil.sol

```diff
@@ -56,8 +54,8 @@ library ChainSpecificUtil {
       uint256 l1BaseFeeScalar = L1BLOCK.baseFeeScalar();
       uint256 l1BlobBaseFeeWei = L1BLOCK.blobBaseFee();
       uint256 l1BlobBaseFeeScalar = L1BLOCK.blobBaseFeeScalar();
-      uint256 weightedGasPrice = (l1BaseFeeScalar * l1BaseFeeWei + l1BlobBaseFeeScalar * l1BlobBaseFeeWei) / 10 ** 6;
-      return weightedGasPrice * (dataSizeBytes + OP_L1_FEE_DATA_PADDING_SIZE);
+      uint256 weightedGasPrice = (16 * l1BaseFeeScalar * l1BaseFeeWei + l1BlobBaseFeeScalar * l1BlobBaseFeeWei) / (16 * 10 ** 6);
```

Comment:
regarding `(16 * 10 ** 6)`, the extra 16 is not needed here right? the weighted gas price is already gas per size_in_byte according to OP official doc